src/main/java/uk/gov/ons/ctp/common/rest/RestClient.java

```diff
@@ -174,43 +177,45 @@ public RestTemplate getRestTemplate() {
     HttpEntity<P> httpEntity = createHttpEntity(objToSend, headerParams);
     UriComponents uriComponents = createUriComponents(path, queryParams, pathParams);
     ResponseEntity<T> response;
+    String errorMessage = "request failed for the given path";
```

Comment:
Having a common message will mean that we will not know exactly which usage below put it in the logs. Please make each log statement have a context specific reason ie which path in the code it came from ie : "Dealing with NOT_FOUND"

Reply:
addressed